experiments/prediction/utils/errors.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+def buildRMSPBE(X, P, R, D, gamma):
```

Comment:
Maybe a reference to Adam's thesis to the specific page that he does this helps?

Reply:
Hmm that's a good question. On one hand, he didn't invent the MSPBE nor this way of computing it. On the other hand, maybe it is a useful reference for how to compute it. I'll think about this. I'm leaning towards saying that anyone looking this deep in the code should expect to see some math and the MSPBE is some pretty straight-forward algebra